gammapy/datasets/tests/test_map.py

```diff
@@ -930,6 +934,7 @@ def test_stack(sky_model):
     assert_allclose(stacked.mask_safe.data.sum(), 4600)
     assert_allclose(stacked.mask_fit.data.sum(), 4600)
     assert_allclose(stacked.exposure.data.sum(), 1.6e11)
+    assert_allclose(stacked._livetime_map.data.sum(), 240000.0)
 
     assert_allclose(stacked.meta_table["OBS_ID"][0], [0, 1])
 
```

Comment:
One could profit of this PR to check if the metadata livetime exists and contains the correct value